tests/testutils/httputils.go

```diff
@@ -44,6 +43,8 @@ func HttpServe(addr string, timeout int) error {
 
 	c := make(chan string)
 	go func() {
+		log.Printf("%v: serving on %v\n", hostname, addr)
```

Comment:
It concerns me that we're using this code from two very different callsites. When used from a test, we should not just log random stuff - it should go through t.Log.